data/Dockerfiles/acme/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.6
+
+LABEL maintainer "Andre Peters <andre.peters@servercow.de>"
+
+RUN apk add --update --no-cache \
+      acme-client \
+      docker \
```

Comment:
I was wondering if wouldn't be possible to achieve the same functionality with API calls (curl) then installing docker wouldn't be necessary - https://docs.docker.com/engine/api/get-started/